server/models/customer.js

```diff
@@ -0,0 +1,46 @@
+//customer: jobs: name, description, required skills, 
+//need date, city, job by id, proposals by id
+
+//finds base in index.js
+const Base = require('./index');
+
+const Customer = Base.descriminator('Customer', new mongoose.Schema({
+    jobsNeeded : [
+        {
+            name: {
+                type: String,
+                required: true,   
+            },
+            jobCustomer: {
+                type: String,
+                required: true,
+            },
+            description: {
+                type: String,
+                required: true,
+                minLength: 1,
+                maxLength: 1000,
+                trim: true,
+            },
+            skills: {
```

Comment:
Maybe optional?
My thinking was that skills would be an array of strings that could be typed in on the front end. Suggest we talk with Stephen and Stephanie on how they want to handle on the front end. App could offer a set list of skills to choose from.